.circleci/config.yml

```diff
@@ -419,12 +420,15 @@ jobs:
           working_directory: api
       - run:
           name: Running tests
-          command: >
-            RUN_ENV=tests venv/bin/pytest << parameters.pytest_extra_args >> --durations=10
-            --junitxml=test-results/junit.xml
+          command: |
+            TEST_FILES=$(circleci tests glob "tests/**/*.py" | circleci tests split --split-by=timings)
```

Comment:
circleci  remembers automatically the timing of each tests and remap it for the new run ?

Reply:
Thanks to the junit xml that contains the information and is stored as artifact :)